src/components/views/rooms/UserOnlineDot.js

```diff
@@ -38,11 +38,11 @@ const RoomTileOnlineDot = ({userId}) => {
     useEventEmitter(user, "User.currentlyActive", currentlyActiveHandler);
     useEventEmitter(user, "User.presence", currentlyActiveHandler);
 
-    return isOnline ? <span className="mx_RoomTile_online_dot" /> : null;
+    return isOnline ? <span className="mx_UserOnlineDot" /> : null;
 };
 
-RoomTileOnlineDot.propTypes = {
+UserOnlineDot.propTypes = {
```

Comment:
imo unless we're using hooks or something that demands the use of a function-based component, we should be using classes. It's kinda important to be able to see the properties of the component at the top of the file.

Reply:
This *is* using hooks though

Reply:
well I look dumb